### PR TITLE
Test formatting and adding this doccumentation

### PR DESCRIPTION
**Pull request by JhanSrbinovsky**
# CABLE

Thank you for submitting a pull request to the CABLE Project.

## Description

I want to verify how this documentation process works and also verify the formatting discrepancies between the documentation which we already have on the TRAC.

Discussion in a  breakout room, involving Claire as well, may  be required 

You can link issues by using a supported keyword in the pull request's description or in a commit message:

Fixes #(issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix
- [x] New or updated documentation

## Checklist

- [x] The new content is accessible and located in the appropriate section.
- [x] I have checked that links are valid and point to the intended content.
- [x] I have checked my code/text and corrected any misspellings

Please add a reviewer when ready for review.

### CODE REVIEW

**Review by rkutteh — COMMENTED** [review]
md syntax with URL needs correcting as suggested and one typo needs fixing as suggested. Also noticed extra spaces between some words.

File: documentation/docs/user_guide/tutorial/uber_quick_guide.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+The CABLE executable (./cable) is configured via the namelist cable.nml. By default the provided cable.nml points to 4-year meteorological forcing from the [http://sites.fluxdata.org/AU-Tum/ Tumbarumba] fluxnet site. NB: spinning up the model has been switched off in cable.nml (spinup=.FALSE.)
+
+We say the "serial" version as CABLE does support a  parralel configuration as well.
+
+ACCESS-ESM1.5 uses  a version of CABLE based on CABLE-2.3.4.
+
+ACCESS-CM2  uses  a version of CABLE that is closer to the HEAD of the trunk at the time of writing (December 2019).
+
+!!For a more detailed discussion of CABLE offline please see [https://trac.nci.org.au/trac/cable/wiki/CableUserGuide this page] from the User Guide.
```
[the remark this answers is not in the document]
Comment (JhanSrbinovsky) [reply documentation/docs/user_guide/tutorial/uber_quick_guide.md]: Manually edited in breakout room with Claire

[the remark this answers is not in the document]
Comment (JhanSrbinovsky) [reply documentation/docs/user_guide/tutorial/uber_quick_guide.md]: although check round brackets situation

File: documentation/docs/user_guide/tutorial/uber_quick_guide.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+{{{
+./cable
+}}}
+
+
+=== In slightly more detail ===
+
+The CABLE executable (./cable) is configured via the namelist cable.nml. By default the provided cable.nml points to 4-year meteorological forcing from the [http://sites.fluxdata.org/AU-Tum/ Tumbarumba] fluxnet site. NB: spinning up the model has been switched off in cable.nml (spinup=.FALSE.)
+
+We say the "serial" version as CABLE does support a  parralel configuration as well.
```
[the remark this answers is not in the document]
Comment (JhanSrbinovsky) [reply documentation/docs/user_guide/tutorial/uber_quick_guide.md]: outdated

**Comment by JhanSrbinovsky** [comment]
can  you have another start this please Ramzi

**Review by rkutteh — APPROVED** [review]

**Pull request merged**